### PR TITLE
DBZ-7917: Document PostgreSQL read-only incremental snapshot.

### PR DESCRIPTION
**Pull request by mfvitale**
Follow up of : https://github.com/debezium/debezium/pull/5611

### CODE REVIEW

File: documentation/modules/ROOT/pages/connectors/postgresql.adoc [hunk 1]
```diff
@@ -259,6 +259,25 @@ include::{partialsdir}/modules/all-connectors/proc-stopping-an-incremental-snaps
 
 include::{partialsdir}/modules/all-connectors/proc-stopping-an-incremental-snapshot-kafka.adoc[leveloffset=+1]
 
+ifdef::community[]
+[id="postgresql-read-only-incremental-snapshots"]
+==== Read-only incremental snapshots
+
+The PostgreSQL connector allows for running incremental snapshots with a read-only connection to the database.
+To run an incremental snapshot with read-only access, the connector uses the current in-progress transaction as high and low watermarks.
+The state of a chunk's window is updated by comparing the txId of write-ahead-log events or the heartbeats against low and high watermarks.
```
Comment (roldanbob) [inline documentation/modules/ROOT/pages/connectors/postgresql.adoc:268]:
```suggestion
The state of a chunk's window is updated by comparing the transaction ID of write-ahead-log events or heartbeat events against low and high watermarks.
```

File: documentation/modules/ROOT/pages/connectors/postgresql.adoc [hunk 2]
```diff
@@ -259,6 +259,25 @@ include::{partialsdir}/modules/all-connectors/proc-stopping-an-incremental-snaps
 
 include::{partialsdir}/modules/all-connectors/proc-stopping-an-incremental-snapshot-kafka.adoc[leveloffset=+1]
 
+ifdef::community[]
+[id="postgresql-read-only-incremental-snapshots"]
+==== Read-only incremental snapshots
+
+The PostgreSQL connector allows for running incremental snapshots with a read-only connection to the database.
```
Comment (roldanbob) [inline documentation/modules/ROOT/pages/connectors/postgresql.adoc:266]:
```suggestion
You can configure a PostgreSQL connector that has a read-only connection to the database to run incremental snapshots.
```
I'm in the process of updating the MySQL/MariaDB docs, and will apply similar wording.

File: documentation/modules/ROOT/pages/connectors/postgresql.adoc [hunk 3]
```diff
@@ -3646,6 +3665,12 @@ After the snapshot completes, this entry is removed.
 No entry is created for the signal to close the snapshot window.
 Set this option to prevent rapid growth of the signaling data collection.
 
+ifdef::community[]
+|[[postgres-property-read-only]]<<postgres-property-read-only, `+read.only+`>>
+|`false`
+|Switch to alternative incremental snapshot watermarks implementation to avoid writes to signal data collection
```
Comment (roldanbob) [inline documentation/modules/ROOT/pages/connectors/postgresql.adoc:3671]:
```suggestion
|Specifies whether a connector writes watermarks to the signal data collection to track the progress of an incremental snapshot.
Set the value to `true` to enable a connector that has a read-only connection to the database to use an incremental snapshot watermarking strategy that does not require writing to the signal data collection.
```
I've applied the same wording to the MySQL/MariaDB docs.

**Review by roldanbob — CHANGES_REQUESTED** [review]
I've suggested a couple of changes. As it happens, I'm in the process of updating similar content elsewhere, so this was good timing in terms of keeping descriptions in sync.

File: documentation/modules/ROOT/pages/connectors/postgresql.adoc (hunk 2 above)
Comment (mfvitale) [reply documentation/modules/ROOT/pages/connectors/postgresql.adoc:266]: I just update it adding 'without requiring the signal data collection table.' WDYT?

Comment (roldanbob) [reply documentation/modules/ROOT/pages/connectors/postgresql.adoc:266]: Changed it to "a signal data collection table". I think that works, si?

**Review by roldanbob — CHANGES_REQUESTED** [review]
Minor update to the one comment. If that looks good to you, I think this is ready to go. Thanks!

**Comment by mfvitale** [comment]
> Minor update to the one comment. If that looks good to you, I think this is ready to go. Thanks!

Thank you Bob!

**Review by roldanbob — APPROVED** [review]

**Comment by jpechane** [comment]
@mfvitale @roldanbob Applied, thanks!

**Pull request merged**